src/ImageList/index.js

```diff
@@ -53,10 +62,9 @@ export default class ImageList extends Component {
     return columns
   }
 
-  renderMasonry() {
+  renderMasonry(columns) {
```

Comment:
This here is instead going to be `items`, not columns. You'll need to send the filtered items through the `this.getColumns` to get the filtered list.